code/game/machinery/computer/arcade.dm

```diff
@@ -397,6 +406,20 @@ GLOBAL_LIST_INIT(arcade_prize_pool, list(
 	var/gameStatus = ORION_STATUS_START
 	var/canContinueEvent = 0
 
+	var/obj/item/radio/Radio
+	var/list/gamers = list()
```

Comment:
more descriptive name please.  `antisocial_gamers` perhaps?

Reply:
I thought we weren't supposed to use redundant variable names